rootex/script/interpreter.cpp

```diff
@@ -50,6 +50,7 @@
 #include "core/resource_files/text_resource_file.h"
 #include "core/resource_files/particle_effect_resource_file.h"
 #include "event_manager.h"
+#include "components/space/transform_animation_component.h"
```

Comment:
Add this line above, where all other component header files are being included. (Line 29)